src/lib/loadCSV.py

```diff
@@ -19,7 +19,10 @@ def upload_csv(csv_file):
     """
     reader = csv.DictReader(csv_file)
 
-    models.set_applications_model(reader.fieldnames) # reset the database
+    models.set_applications_model(reader) # reset the database
+
+    csv_file.seek(0)
+    next(reader)
     
     for row in reader:
         _store_csv_row(row)
```

Comment:
Can we pass in an instance of ApplicationsModel here instead of instantiating one for every row?

Reply:
I'll just inline _store_csv_row